src/schema/v2/me/createCollectionMutation.ts

```diff
@@ -87,3 +87,24 @@ export const createCollectionMutation = mutationWithClientMutationId<
     }
   },
 })
+
+const formatGravityHttpError = (error) => {
```

Comment:
One thought about that is that if it's exported, it's public. If this refactor work somehow is put down then there's a risk that this might spread. 

That's kinda the central point about this given the volume of use that this particular function gets and its critical importance in the stack. If it's local, its like: we use it here for now to get over the hump.

With that said, I do think a `// TODO` or `// FIXME` is appropriate here, as a reminder.

Reply:
Yeah, true that VS Code or other tooling might pick up on these once they're exported, and make it too easy for these to proliferate.